packages/react-core/src/components/MenuToggle/MenuToggle.tsx

```diff
@@ -97,6 +97,7 @@ class MenuToggleBase extends React.Component<MenuToggleProps> {
             aria-expanded={isExpanded}
             onClick={onClick}
             aria-label={ariaLabel || 'Menu toggle'}
+            tabIndex={-1}
```

Comment:
Was the main issue with being able to focus and click the toggle via keyboard the focus/styling issue?

Just wondering if it'd be confusing that this toggle can be interacted with via mouse and keyboard when using a screen reader, but not via keyboard only.

Reply:
The main issue was that it focused the first menu item after focusing and clicking the toggle with keyboard, so then user is not able to write text to the input. We always want to have the input focused I think.

I am checking the w3 combobox example for reference: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-list/ and it also has a `tabindex="-1"`

Reply:
Sounds good, we can keep this in and see if anyone brings it up and revisit it then if so.

Changing the toggle icon would probably be worth a discussion since it'd have to include updating anywhere else we're using icons in a similar fashion, as well as making sure we have documentation about consumers needing to do the same if they're doing something more custom.